src/main/java/lotto/domain/PurchaseMoney.java

```diff
@@ -0,0 +1,67 @@
+package lotto.domain;
+
+import java.util.Objects;
+
+import lotto.exception.IllegalMoneyUnitException;
+
+public class PurchaseMoney {
+	public static final int LOTTO_PIECE_PRICE = 1000;
+	public static final int MIN = 1000;
+	public static final int MAX = 100000;
```

Comment:
```suggestion
	public static final int MAX = 100_000;
```

언더스코어를 추가하여 가독성을 상승시킬 수 있습니다.